language-support/ts/daml-react/credentials.ts

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2020 The DAML Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { decode } from 'jwt-simple';
+
+const LEDGER_ID: string = 'DAVL';
```

Comment:
This should probably go away before we publish this as a library.

Reply:
good catch, thanks!